Bugherd # 698 - Removing reference to Esper (#1126)
SoftwareAG/c8y-docs: content/users-guide/cockpit-bundle/smart-rules.md

- [Administration > Managing business rules](/users-guide/administration#business-rules) for details on managing smart rules for your devices.
{{< /c8y-admon-related >}}

> {{< product-c8y-iot >}} includes a rule engine to analyze data in realtime and to perform actions based on data.

Reviewer: Realising that "rule engine" isn't a good description for the streaming analytics component:
```suggestion
{{< product-c8y-iot >}} includes Streaming Analytics which can analyze data in realtime and perform actions based on data.
```

Reviewer: You might also want to xref to the Streaming Analytics guide.

Reviewer 2: I'd put it as "... includes the Streaming Analytics application which ...."

> {{< product-c8y-iot >}} includes Streaming Analytics which can analyze data in realtime and perform actions based on data.

Reviewer 2: ```suggestion
{{< product-c8y-iot >}} includes the Streaming Analytics application which can analyze data in realtime and perform actions based on data.
```
